packages/components/src/components/Card/utils.tsx

```diff
@@ -0,0 +1,28 @@
+import { spacingsPx } from '@trezor/theme';
+import { PaddingType } from './types';
+
+type MapArgs = {
+    $paddingType: PaddingType;
+};
+
+export const mapPaddingTypeToLabelPadding = ({ $paddingType }: MapArgs): number | string => {
+    const paddingMap: Record<PaddingType, number | string> = {
+        none: `${spacingsPx.xxs} 0`,
+        small: `${spacingsPx.xxs} ${spacingsPx.sm}`,
+        normal: `${spacingsPx.xs} ${spacingsPx.lg}`,
+        large: `${spacingsPx.sm} ${spacingsPx.xl}`,
+    };
+
+    return paddingMap[$paddingType];
+};
+
+export const mapPaddingTypeToPadding = ({ $paddingType }: MapArgs): number | string => {
+    const paddingMap: Record<PaddingType, number | string> = {
+        none: 0,
+        small: spacingsPx.sm,
+        normal: spacingsPx.lg,
+        large: spacingsPx.xl,
```

Comment:
normal -> lg padding
large -> xl padding

How about 
having normal, large and xlarge with proper spacings

Reply:
Sorry, I don't understand the point:)

Reply:
we call paddingType `normal` but we use `lg` spacing.

Reply:
As consumers of the component, we don't care about the right side of the mapping function, that's just internal design system token naming:) For this component, normal means such and such padding. Hope it makes sense:)